index.js

```diff
@@ -1,23 +1,66 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
+const fs = require("fs");
 
 // Import of the model Recipe from './models/Recipe.model.js'
-const Recipe = require('./models/Recipe.model');
+const Recipe = require("./models/Recipe.model");
 // Import of the data from './data.json'
-const data = require('./data');
+const data = require("./data");
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+const MONGODB_URI = "mongodb://localhost:27017/recipe-app";
+
+const recipesData = JSON.parse(fs.readFileSync(__dirname + '/data.json', 'utf-8'));
+
+// async function startLab() {
+//   await mongoose.connect(MONGODB_URI);
+//   await mongoose.connection.db.dropDatabase();
+//   await mongoose.connection.close();
+// }
+
+// startLab();
+
+const recipe1 = new Recipe({
+  title: "R1",
+  level: "Easy Peasy",
+  // enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+  // ,ingredients:
+  cuisine: "Spanish",
+  dishType: "other",
+  // enum: ["breakfast","main_course","soup","snack","drink","dessert","other",],
+  // ,image:
+  // ,duration:
+  // ,creator:
+  // ,created:
+});
 
 // Connection to the database "recipe-app"
 mongoose
   .connect(MONGODB_URI)
-  .then(x => {
+  .then((x) => {
     console.log(`Connected to the database: "${x.connection.name}"`);
     // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+    return Recipe.deleteMany();
   })
-  .then(() => {
+  .catch((error) => {
+    console.error("Error connecting to the database", error);
+  });
+
+mongoose
```

Comment:
this is somewhat broken. 
you are calling mongoose.connect twice - and you don't properly wait for the first connection to be done deleting all there is in the database. this could lead to possible raceconditions. instead either -> use async and await or do whatever you need to do within chained .then blocks after the return Recipe.deleteMany()